### PR TITLE
Add thumbnails icons in resources list

### PR DESCRIPTION
**Pull request by Bouh**
From #1724

image

### CODE REVIEW

File: newIDE/app/src/ResourcesList/index.js [hunk 1]
```diff
@@ -155,6 +155,25 @@ export default class ResourcesList extends React.Component<Props, State> {
...
+      case 'json':
+        return 'res/actions/fichier24.png';
+      case 'video':
+        return 'JsPlatform/Extensions/videoicon24.png';
+      case 'font':
+        return 'res/actions/font24.png';
+      case 'bitmapFont':
+        return 'JsPlatform/Extensions/bitmapfont32.png';
+      default:
+        return 'res/unknown32.png'; //Icon type inconnu
```
Comment (4ian) [inline newIDE/app/src/ResourcesList/index.js:173]: Comment in French. Remove it entirely, the "default" is already telling what's going on.

File: newIDE/app/src/ResourcesList/index.js [hunk 2]
```diff
@@ -155,6 +155,25 @@ export default class ResourcesList extends React.Component<Props, State> {
     );
   };
 
+  _getResourceThumbnail = (resource: gdResource) => {
+    switch (resource.getKind()) {
+      case 'image':
+        return getLocalResourceFullPath(this.props.project, resource.getName());
```
Comment (4ian) [inline newIDE/app/src/ResourcesList/index.js:161]: The name is not necessarily the file path. Use getFile

Comment (4ian) [reply newIDE/app/src/ResourcesList/index.js:161]: This is using getLocalResourceFullPath - is this electron only?
What is happening in the web app when this is called?

**Review by 4ian — COMMENTED** [review]
Good idea to add this!

**Comment by Bouh** [comment]
Fixes are made.
I've open examples and tested all types of objects, on desktop and web app.
Even the default icon on broken resources.
image

**Review by 4ian — APPROVED** [review]
👍

**Pull request merged**